platform/podjs/src/browserPod.ts

```diff
@@ -300,28 +303,51 @@ class PodJsInfo implements Info {
     }
 }
 
+interface NetworkResponse {
+    payload?: string;
+    responseCode: number | undefined;
+}
+interface Network {
+    httpPost(
+        url: string,
+        body: string,
+        contentType?: string,
+        authorization?: string
+    ): Promise<NetworkResponse>;
+    httpGet(
+        url: string,
+        contentType?: string,
+        authorization?: string
+    ): Promise<NetworkResponse>;
+}
+
 class BrowserNetwork implements Network {
```

Comment:
I don't think you need an interface for this any longer now, it's just an internal class really.